api/v1beta1/packetcluster_webhook.go

```diff
@@ -47,6 +47,23 @@ func (c *PacketCluster) Default() {
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type.
 func (c *PacketCluster) ValidateCreate() error {
 	clusterlog.Info("validate create", "name", c.Name)
+	allErrs := field.ErrorList{}
+
+	// Must have either one of Metro or Facility
+	if len(c.Spec.Facility) == 0 && len(c.Spec.Metro) == 0 {
+		allErrs = append(allErrs,
+			field.Invalid(field.NewPath("spec", "Metro"),
+				c.Spec.Metro, "field is required when Facility is not set"),
+		)
+	}
+
+	// If both Metro and Facility are set, ignore Facility, we'll leave this to
```

Comment:
The comment says ignore facility if metro is also set, but it doesn't look like facility is ignored here.  Does the comment need to be updated / removed / moved?